spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -261,7 +265,9 @@ Spell Actions (Per Exec Doc):
     * [ ] Upgradable SubDAO contracts
       * [ ] Any upgradable contracts have the `PAUSE_PROXY` as their `admin` (i.e. the party that can upgrade)
         * [ ] Any upgradable SubDAO contracts with an `admin` that is not `PAUSE_PROXY` are not authed on any core contracts (Blocking)
-    * [ ] All SubDAO content addresses (i.e. provided contract addresses or EOAs) present in the Maker Core spell are present in the Exec Doc and are correct. SubDAO addresses being authed or given any permissions MUST be in the Exec Doc. SubDAO addresses being called must be confirmed by the SubDAO spell team.
+    * [ ] All SubDAO content addresses (i.e. provided contract addresses or EOAs) present in the Maker Core spell are present in the Exec Doc and are correct.
+      * [ ] SubDAO addresses being authed or given any permissions MUST be in the Exec Doc.
+      * [ ] SubDAO addresses being called must be confirmed by the SubDAO spell team.
```

Comment:
In the future, please try to avoid sneaking more changes in a single PR, let's stick to a single-purpose PRs. The reasons are:
- Clean history and git blame
- In the mixed setup it's easier to miss some inconsistencies as the changes are of different types
- Impossible to prioritise 1-line change that require 10 minutes to review over 20 other changes that require 1 minute each

Reply:
I'll revert this.